packages/webdriver/src/types.ts

```diff
@@ -240,7 +240,7 @@ export interface Capabilities {
 
 export interface W3CCapabilities {
     alwaysMatch: Capabilities;
-    firstMatch: Capabilities;
+    firstMatch: Capabilities[];
```

Comment:
Is this a list of a list of capabilities?

Reply:
yes